app/routes.js

```diff
@@ -4,22 +4,33 @@
         .module('weatherApp')
         .config(config);
 
-    function config($routeProvider) {
-        $routeProvider
-            .when('/', {
-                templateUrl: 'home/home.html',
-                controller: 'homeController',
-                controllerAs: 'vm'
-            })
-            .when('/forecast', {
-                templateUrl: 'forecast/forecast.html',
-                controller: 'forecastController',
-                controllerAs: 'vm'
-            })
-            .when('/forecast/:days', {
-                templateUrl: 'forecast/forecast.html',
-                controller: 'forecastController',
-                controllerAs: 'vm'
-            });
+    function config($stateProvider) {
```

Comment:
You can actually chain state definitions, similarly to $routeProvider, if you wanted to keep the same format. Just a matter of preference.

Example:
```
$stateProvider
    .state('home', {
        url: '/',
        ...
    })
    .state(...)
    .state(...)
    ...
```